src/dev/typescript/projects.ts

```diff
@@ -42,5 +42,5 @@ export function filterProjectsByFlag(projectFlag?: string) {
   }
 
   const tsConfigPath = resolve(projectFlag);
-  return PROJECTS.filter(project => project.tsConfigPath === tsConfigPath);
+  return PROJECTS.filter(project => project.tsConfigPath.includes(tsConfigPath));
```

Comment:
I found it useful to be able to run tslint for x-pack like this:
`node scripts/tslint --project x-pack`

Is that okay, or should I revert this change?

Reply:
I'd be fine with something a little bit stricter, maybe:

```
project.tsConfigPath = tsConfigPath || project.tsConfigPath === resolve(tsConfigPath, 'tsconfig.json')
```

Pretty sure as is `--project .` will match any `tsconfig.json` file, depending on which one is first.